README.md

```diff
@@ -147,14 +147,20 @@ variable name.
 
 ### Commit messages
 
-Here is a generally nice guide to how to write good commit messages:
+Here is a nice guide on how to write good commit messages:
 https://chris.beams.io/posts/git-commit/
 
 So sum it up the most important parts are:
 
 * Capitalize the subject line.
 * Do not end the subject line with a period.
-* Use the imperative mood in the subject line.
+* Use the imperative mood in the subject line, starting with a verb.
+* Limit the subject line to 72 characters*
```

Comment:
The action seems to limit the [body](https://github.com/mullvad/mullvadvpn-app/actions/runs/8739630001/job/23981376204?pr=6145) to 72 chars as well.

Reply:
Yes, but that's already written out in https://chris.beams.io/posts/git-commit/. The subject line length is *different* from the article we link to, so more important to point out.

The list is also titled *the most important parts* and not a complete list. IMO the body line length is not among the top important rules to follow.

Reply:
We could of course make the list more extensive. And keep the link to the post around for a more verbose version and more motivation as to *why* the rules are there. But I decided not to do that, at least not now.